ext/xsl/xsltprocessor.c

```diff
@@ -21,61 +21,9 @@
 
 #include "php.h"
 #include "php_xsl.h"
+#include "php_xsl_arginfo.h"
```

Comment:
As it's only used in this file now, please rename to `xsltprocessor.stub.php` (makes the make file happier).